Nodejs/Product/Nodejs/Project/NodejsProjectLauncher.cs

```diff
@@ -280,7 +288,7 @@ private void StartWithChromeV2Debugger(string file, string nodePath, bool should
             var debugTargets = new[] {
                 new VsDebugTargetInfo4() {
                     dlo = (uint)DEBUG_LAUNCH_OPERATION.DLO_CreateProcess,
-                    guidLaunchDebugEngine = WebKitDebuggerV2Guid,
+                    guidLaunchDebugEngine = ShouldUseV3CdpDebugger() ? JsCdpDebuggerV3Guid : WebKitDebuggerV2Guid,
```

Comment:
Maybe the helper should just return the engine GUID?

Reply:
Done!